### PR TITLE
LPS-88715 Add integration tests for adaptive media

### PR DESCRIPTION
**Pull request by ruben-pulido**

### CODE REVIEW

**Comment by ruben-pulido** [comment]
ci:test:sf <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ruben-pulido** [comment]
ci:test:relevant <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java [hunk 1]
```diff
@@ -91,22 +106,71 @@ public void testAddMediaObject() throws Exception {
...
+			_getAdaptiveMedias(fileEntry);
+
+		Assert.assertEquals(
+			adaptiveMediaList.toString(), 2, adaptiveMediaList.size());
+
+		int numberOfBytes = bytes.length;
+
+		_verifyAdaptiveMedia(
+			adaptiveMediaList.get(0), 512, 22606L, "Preview-1000x0",
+			numberOfBytes);
```
Comment (cgoncas) [inline modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java]: use directly bytes.length

File: modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java [hunk 2]
```diff
@@ -91,22 +106,71 @@ public void testAddMediaObject() throws Exception {
...
+
+		_verifyAdaptiveMedia(
+			adaptiveMediaList.get(1), 300, 12598L, "Thumbnail-300x300",
+			numberOfBytes);
```
Comment (cgoncas) [inline modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java]: use directly bytes.length

File: modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java [hunk 3]
```diff
@@ -213,6 +299,43 @@ private String _getFileEntryPreviewURL(FileEntry fileEntry)
 			getNestedCollectionResource(), fileEntry);
 	}
 
+	private void _verifyAdaptiveMedia(
```
Comment (cgoncas) [inline modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java]: we should call this method assert, could we create a Matcher instead to use a generic method?

File: modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java [hunk 4]
```diff
@@ -137,11 +203,13 @@ public void testGetMediaObjectAssetTags() throws Exception {
...
 		String content = RandomTestUtil.randomString(10);
 
+		byte[] bytes = content.getBytes(StandardCharsets.UTF_8);
+
 		_addFileEntry(
 			_group.getGroupId(),
 			new MediaObjectImpl(
 				new BinaryFile(
-					new ByteArrayInputStream(content.getBytes(UTF_8)), 0L,
+					new ByteArrayInputStream(bytes), 0L,
```
Comment (cgoncas) [inline modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java]: if bytes is used only here. don't declare the variable and use it here: content.getBytes(StandardCharsets.UTF_8)

File: modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java [hunk 5]
```diff
@@ -115,11 +179,13 @@ public void testGetMediaObjectAssetTags() throws Exception {
...
 		List<String> keywords = Arrays.asList("keyword1", "keyword2");
 
+		byte[] bytes = content.getBytes(StandardCharsets.UTF_8);
+
 		FileEntry fileEntry = _addFileEntry(
 			_group.getGroupId(),
 			new MediaObjectImpl(
 				new BinaryFile(
-					new ByteArrayInputStream(content.getBytes(UTF_8)), 0L,
+					new ByteArrayInputStream(bytes), 0L,
```
Comment (cgoncas) [inline modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java]: if bytes is used just here. replace it for  content.getBytes(StandardCharsets.UTF_8)

**Pull request closed without merging**